test/remote/account_signin_verification_tests.js

```diff
@@ -447,6 +447,68 @@ TestServer.start(config)
     )
 
     test(
+      'certificate sign with keys=false session',
+      function (t) {
+        var email = server.uniqueEmail()
+        var password = 'allyourbasearebelongtous'
+        var client = null
+
+        return Client.createAndVerify(config.publicUrl, email, password, server.mailbox, {keys:true})
```

Comment:
Yes, there's a `client.login({keys:false})` further down that's the one actually under test; I can add a comment to clarify though